### ISSUE TITLE
SC 2.5.5 Understanding Doc has a typo (WCAG 2.1)

### ISSUE THREAD

**Issue by gjccopinga** [w3c/wcag#2112]
Hello,

In the Understanding Doc for SC 2.5.5 (https://www.w3.org/WAI/WCAG21/Understanding/target-size.html) there is a typo in both failure techniques. They refer to "2.5.3". This should be "2.5.5".

The same problem occurs in the "How to meet" document (https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=255#target-size)
